oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1700 @@
+{
```

Comment:
<p>The dataframe should be aligned in the same way as in <code>alive_or_not</code>  dataframe to get a good representation of the sex proportions in different classes (with pie chart). In the case of your dataframe you can leave the bar chart as it is or try to fix the dataframe to use it with pie.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>